conf/messages

```diff
@@ -100,14 +100,10 @@ openPayments.incorrect = If what you owe is incorrect, check if you can
 openPayments.correctErrors = correct errors on your VAT Return (opens in a new tab)
 openPayments.whatOweHelp.heading = What I owe is incorrect or missing
 openPayments.directDebitAlreadySetUp = If you have already set up a Direct Debit, you do not need to pay now.
```

Comment:
Can you also remove these other "direct debit" messages as they are now unused. If you want to confirm if a message is unused, do a global find (Cmd + Shift + F) for the message key and if the only matches are in the english and welsh `messages` files then it's unused.